acceptance/install/scenario7_test.go

```diff
@@ -0,0 +1,142 @@
+package install_test
+
+import (
+	"encoding/json"
+	// "fmt"
+	"os"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/epinio/epinio/acceptance/helpers/catalog"
+	"github.com/epinio/epinio/acceptance/helpers/epinio"
+	"github.com/epinio/epinio/acceptance/helpers/proc"
+	"github.com/epinio/epinio/acceptance/testenv"
+)
+
+var _ = Describe("<Scenario7> Rancher Desktop install and push epinio on self hosted runner", func() {
+	var (
+		flags             []string
+		epinioHelper      epinio.Epinio
+		configurationName = catalog.NewConfigurationName()
+		appName           string
+		loadbalancer      string
+		registryUsername  string
+		registryPassword  string
+		// rangeIP           string
+		domain            string
+		domainIP          string
```

Comment:
this fails both when is commented and when it is not.
As I undestand it `domainIP` is being used later in line 45 `domainIP = strings.TrimSuffix(domain, ".omg.howdoi.website")`; however I get [this](https://github.com/epinio/epinio/runs/7766953484?check_suite_focus=true) error:

```
# github.com/epinio/epinio/acceptance/install_test
Error: vet: acceptance/install/scenario7_test.go:29:3: domainIP declared but not used
```
I am not sure why. @enrichman would you know?

Reply:
Spoke offline about it.
I am commenting both lines out